vdr/didweb/manager.go

```diff
@@ -263,7 +238,7 @@ func (m Manager) createService(tx *gorm.DB, subjectDID did.DID, service did.Serv
 
 	_, err = didDocumentManager.CreateOrUpdate(current.DID, current.VerificationMethods, append(current.Services, sqlService))
 
-	return &service, nil
+	return &service, err
```

Comment:
missing test?

Reply:
probably, but will be removed in next PR anyway